contrib/init/bitcoind.service

```diff
@@ -11,7 +11,11 @@
 
 [Unit]
 Description=Bitcoin daemon
-After=network.target
+Documentation=https://github.com/bitcoin/bitcoin/blob/master/doc/init.md
```

Comment:
Is there a way to cleanly link to the local file instead of github?

Reply:
idk about such a way.
Have you any concerns about a link to the file in the github repo? If so, I could drop this change.